radixdlt/src/main/java/org/radix/api/http/RadixHttpServer.java

```diff
@@ -362,6 +371,51 @@ public final JSONObject disconnectAllPeers() {
 		return result;
 	}
 
+	/**
+	 * Handle PUT request for changing BFT state.
+	 * <p>
+	 * Put request takes two parameters:
+	 * <ul>
+	 *   <li><b>id</b> the ID of the BFT instance (must be {@code 0} for now)</li>
+	 *   <li><b>state</b> {@code true} to enable the specified instance id, otherwise
+	 *     the instance is disabled
+	 * </ul>
+	 *
+	 * @param exchange The {@link HttpServerExchange} to use
+	 * @throws IOException if an error occurs parsing form data
+	 */
+	private void handleBftState(HttpServerExchange exchange) throws IOException {
+		try (final FormDataParser formDataParser = this.formParserFactory.createParser(exchange)) {
```

Comment:
Given that all of our apis use a JSON body, I think we should stay consistent and not introduce a form-data body.